qiskit/extensions/standard/ryy.py

```diff
@@ -0,0 +1,77 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2020.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""
+Two-qubit YY-rotation gate.
+"""
+import numpy as np
+from qiskit.circuit import Gate
+from qiskit.circuit import QuantumCircuit
+from qiskit.circuit import QuantumRegister
+
+
+class RYYGate(Gate):
+    """Two-qubit YY-rotation gate.
+
+    This gate corresponds to the rotation U(θ) = exp(-iθ/2 Y⊗Y), multiplied with
+    the phase exp(iθ/2).
```

Comment:
I would probably use sphinx math mode for this part of the docstring

Reply:
Does sphix math support latex notation? Meaning 
```rst
This gate corresponds to the rotation :math:`U(\theta) = e^{-i\theta / 2 Y \otimes Y)`, 
multiplied with the phase :math:`e^{i \theta / 2}`
```
would be the correct way?

Reply:
Yes you can put any standard latex in `:math:`. If you use backslashes though you either need to make the doc a raw string `r"""` or use a double backslash for all tex commands.

Reply:
Ok, thanks!